fleetspeak/src/server/internal/services/manager.go

```diff
@@ -78,6 +78,21 @@ func NewManager(dataStore db.Store, serviceRegistry map[string]service.Factory,
 	return &m
 }
 
+// getClientData returns client data corresponding to client that is the source of the given message.
+func (c *Manager) getClientData(ctx context.Context, m *fspb.Message) (*db.ClientData, error) {
```

Comment:
Just `ClientData` should suffice.

https://google.github.io/styleguide/go/decisions.html#getters